WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -556,6 +556,13 @@ + (NSString *)timeZone
     ? [elementCache elementForUUID:(NSString *)request.parameters[@"uuid"]]
     : request.session.activeApplication;
   id keys = request.arguments[@"keys"];
+
+  if (![destination respondsToSelector:@selector(typeKey:modifierFlags:)]) {
+    NSString *message = @"Not supported on this OS version";
```

Comment:
typeKey API is only supported since Xcode17/iPadOS 17

Reply:
d7a75247edfb5a4ef28fc09437ad96513339dbd2